aten/src/ATen/native/layer_norm.cpp

```diff
@@ -69,24 +98,25 @@ Tensor layer_norm(
     double eps,
     bool /* cudnn_enable, deprecated */) {
 
-  auto inputs = _prepare_layer_norm_inputs(input, normalized_shape, weight, bias);
```

Comment:
Previously before dispatching to device we do _prepare_layer_norm_inputs(which does input validation etc) first, but now it has been moved into device kernels. If this is intended you should also add it to math_native_layer_norm as well.